docs/typings-examples.md

```diff
@@ -0,0 +1,86 @@
+# Examples
+Here are some examples based on different shapes of the source module.
+
+## Exporting one function
+
+```ts
+declare function domready (...): any;
+
+export = domready
+```
+https://github.com/typed-typings/npm-domready
+
+## Export function with overloads
+```ts
+declare function xtend <A> (a: A): A;
+declare function xtend <A, B> (a: A, b: B): A & B;
+export = extend;
+```
+https://github.com/typed-typings/npm-xtend
+
+## Exporting function + utility
+
+```ts
+declare function globby (...): any;
+
+declare module globby {
+  function sync (...): any;
+}
+
+export = globby;
+```
+https://github.com/typed-typings/npm-globby
+
+```ts
+declare function tape(): any;
+declare module tape {
+  export function skip(): any;
+}
+
+export = tape;
+```
+https://github.com/typed-typings/npm-tape
+
+## Exporting immutable utility
```

Comment:
What does this header mean? Maybe we can add a description for each section? 

The only difference with `diff` is that it uses the old style `declare module` + `export =` instead of multiple exports.